Apps/WebClient/src/ClientApp/src/models/storeState.ts

```diff
@@ -61,7 +61,8 @@ export interface LaboratoryState {
 }
 
 export interface ImmunizationState {
-    immunizations: ImmunizationModel[];
+    immunizations: ImmunizationEvent[];
+    recomendations: Recomendation[];
```

Comment:
I couldnt find where the recommendations are being stored. From the look of the action.ts, we are only querying if the views are empty; might need to query if the views or the recommendations are empty

Reply:
just added to the same mutation, since this comes from the same endpoint, why would we query it again?